Dehaat/web-app-template
# [IMP]: Integrated storybook

**Closes:** [DP-218](https://agrevolution.atlassian.net/browse/DP-218)

# Why this change?

This change integrates [storybook](https://storybook.js.org) to the web app template. The intention behind integrating the storybook is following

- Independent component development and testing
- Ease of review by just looking at a story and playing around with it
- Ease of sharing the reusable component via storybook
- Provides features of multiple Addon's like accessibility, dark and light modes, multiple screen resolutions, and the option to create custom adding based on requirements
- Nicely integrates with jest and reduces code for snapshot testing

# How to test?
- Clone the repository
- Run `npm install`
- Run `npm run storybook`

Running `npm run storybook` should open the storybook in the browser tab automatically or else you can copy-paste the URL provided by the command in the browser to view the storybook.
